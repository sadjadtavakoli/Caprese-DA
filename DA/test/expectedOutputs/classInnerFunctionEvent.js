
#3 function TextInlines-test/unit_tests/classinnerfunctionevent.js-3-19 entered from classInnerFunctionEvent.js-test/unit_tests/classinnerfunctionevent.js-1-23
#8 function buildInlines-test/unit_tests/classinnerfunctionevent.js-8-18 entered from classInnerFunctionEvent.js-test/unit_tests/classinnerfunctionevent.js-1-23
#11 function  listner1-test/unit_tests/classinnerfunctionevent.js-11-13 entered throught event connection emitted by function buildInlines-test/unit_tests/classinnerfunctionevent.js-8-18
#end Execution