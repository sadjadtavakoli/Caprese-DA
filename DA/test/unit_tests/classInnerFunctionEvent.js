var events = require('events');

class TextInlines {

    constructor(pdfDocument) {
        this.pdfDocument = pdfDocument;
    }
    buildInlines() {
        var eventEmitter = new events.EventEmitter();
        
        function listner1() {
        //   do nothing 
        }
        
        eventEmitter.addListener('connection', listner1);
        
        eventEmitter.emit('connection');
    }
}


let textInlines = new TextInlines("pdfDocument");
textInlines.buildInlines()
