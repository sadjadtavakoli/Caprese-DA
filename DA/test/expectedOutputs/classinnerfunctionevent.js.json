{"TextInlines-test/unit_tests/classinnerfunctionevent.js-3-19":{"impacted":["t_1"]},"test/unit_tests/classinnerfunctionevent.js":{"impacted":["t_7"]},"buildInlines-test/unit_tests/classinnerfunctionevent.js-8-18":{"impacted":["t_1"]},"keyMap":{"t_1":"test/unit_tests/classinnerfunctionevent.js","t_7":"TextInlines-test/unit_tests/classinnerfunctionevent.js-3-19","t_13":"buildInlines-test/unit_tests/classinnerfunctionevent.js-8-18","t_23":"listner1-test/unit_tests/classinnerfunctionevent.js-11-13"}}